include/mbedtls/ssl.h

```diff
@@ -4478,6 +4492,20 @@ void mbedtls_ssl_conf_preference_order(mbedtls_ssl_config *conf, int order);
  *                                         MBEDTLS_SSL_SESSION_TICKETS_DISABLED)
  */
 void mbedtls_ssl_conf_session_tickets(mbedtls_ssl_config *conf, int use_tickets);
+
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3)
+/**
+ * \brief Enable / Disable TLS 1.3 handling of NewSessionTicket messages (client and TLS 1.3 only).
+ *        (Default: MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_DISABLED)
```

Comment:
(Repeating an earlier comment so it doesn't get forgotten) Please document somewhere (here would be a good place) that this defaults off for backward compatibility with non-1.3-aware applications written before 3.6.0. (Similar to e.g. [this](https://github.com/Mbed-TLS/mbedtls/pull/9501/files#diff-b9b23ef0e168bb80241e10e0308a5e7859922a01c0090984b484f2e9f388ce92R1894).)